src/Vault/Vault.csproj

```diff
@@ -28,6 +28,13 @@
   <ItemGroup Condition=" '$(TargetFramework)' == 'net45' ">
     <Reference Include="System.Web" />
     <Reference Include="System.Net.Http" />
+    <Reference Include="System.Net.Formatting" />    
   </ItemGroup>
+  
+  <ItemGroup Condition="'$(TargetFramework)' == 'net45'">
```

Comment:
We are moving away from support specific .NET runtimes to ensure compatibility.  Please target .NET Standard 1.1.  There is another pull request already that removes .NET 4.5 specific support.